virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -282,6 +282,10 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //4.14 leave in per Dan K
 * You cannot run {VirtProductName} on a single-stack IPv6 cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2193267[*BZ#2193267*])
 
+//CNV-38746
+* When you perform a Control Plane Only update from {product-title} 4.12 to an even-numbered minor version, VMs that use the `cnv-bridge` Container Network Interface (CNI) fail to live migrate. (link:https://access.redhat.com/solutions/7069807[])
```

Comment:
```suggestion
* When you perform a Control Plane Only update from {product-title} 4.12 to newer version, VMs that use the `cnv-bridge` Container Network Interface (CNI) fail to live migrate. (link:https://access.redhat.com/solutions/7069807[])
```

I think it is for any upgrade. Should we add this release note to all releases? If somebody plans an upgrade from 4.12 all the way to 4.18, they may miss some release notes.

Once 4.12 is EOL and its documentation is taken offline, we can remove the release note from everywhere

Reply:
I am confused because the Jira description says that it is an [EUS-to-EUS](https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html-single/updating_clusters/index#control-plane-only-update) update.

The way our release notes files are set up, I will need to create separate PRs for each version. I was planning to do that after the content is approved here.

Reply:
Oh okay. How about deleting the reference to EUS update and changing the text to:

> 
When you update  from {product-title} 4.12 to a newer version, VMs that use the `cnv-bridge` Container Network Interface (CNI) fail to live migrate. 
>

Reply:
@phoracek Wll the above phrasing work?